app/js/components/labOrderEntry/LabEntryForm.jsx

```diff
@@ -140,25 +138,6 @@ export class LabEntryForm extends PureComponent {
     return dispatch(deleteDraftLabOrder());
   }
 
-  showFieldSet = () => (
-    <div>
-      <LabPanelFieldSet
-        labCategoryName={this.state.categoryName}
-        handleTestSelection={this.handleTestSelection}
-        panels={this.props.conceptsAsPanels}
-        selectedPanelIds={this.state.selectedPanelIds}
-        locale={this.props.sessionReducer.locale}
-      />
-      <LabTestFieldSet
-        labCategoryName={this.state.categoryName}
-        handleTestSelection={this.handleTestSelection}
-        selectedTests={this.props.selectedTests}
-        tests={this.props.standAloneTests}
-        locale={this.props.sessionReducer.locale}
-      />
-    </div>
-  );
```

Comment:
Yeah, it's easier to read without having these function blocks that just render and aren't reused. 

Are there any other advantages or disadvantages of making this move?

Reply:
(Oh, yeah, I see you referenced this in your overall comments... sounds good to me)

Reply:
No disadvantages I can think of. I think React best practice is that all the JSX lives in a React component's render method (or the return statement of a React function component).